src/pages/Admin/VenueList/VenueList.tsx

```diff
@@ -3,21 +3,23 @@ import { Link } from "react-router-dom";
 
 import { isVenueWithRooms } from "types/venues";
 
-import { orderedVenuesSelector } from "utils/selectors";
 import { canHaveSubvenues } from "utils/venue";
 
-import { useSelector } from "hooks/useSelector";
+import { useRelatedVenues } from "hooks/useRelatedVenues";
```

Comment:
This component is not being imported or used anywhere. `Admin.tsx` is using `VenueList` that's already defined inside it's own file.

Reply:
removed in cebe716d91ee6f1f6f15bcf2cadeecbc2c31a060